app/build.gradle

```diff
@@ -83,6 +83,12 @@ android {
         reactNativeLib {
             dimension "version"
         }
+        advancedDeviceAndroidLib {
+            dimension "version"
+        }
+        advancedDeviceReactNativeLib {
+            dimension "version"
+        }
```

Comment:
Does fingerprint have a min version for android we need to know about? in iOS we had to require min iOS 12

Reply:
According to the docs here: https://github.com/fingerprintjs/fingerprintjs-pro-android-demo it is API 21+ (Android 5) so we should be good.